### PR TITLE
feat: ascatngs hg38 references

### PR DESCRIPTION
**Pull request by khurrammaqbool**
### This PR:
Added: ascatngs hg38 reference files.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -13,6 +13,10 @@ Fixed:
 
 * Fixed context for Dockerfile for release content #720
 
+Added:
+
+* ``ascatngs`` hg38 references
```
Comment (ashwini06) [inline CHANGELOG.rst:18]:
```suggestion
* ``ascatngs`` hg38 references to constants #683 
```

**Review by ashwini06 — APPROVED** [review]
👍

**Review by hassanfa — APPROVED** [review]
minor thing: a bit more verbose changelog note

**Comment by khurrammaqbool** [comment]
Screenshot 2021-08-31 at 10 34 35

**Pull request merged**